website/docs/verifiable-document/comparison.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Comparison
+sidebar_label: Comparison
+---
+
+In this section, we will discuss the differences between Ethereum Smart Contract Verifiable Credentials and DID Verifiable Credentials. The goal is to provide enough information for you to choose the correct workflow depending on your needs.
+
+## Infrastructure
+
+By using Ethereum Smart Contracts, you will of course be bound to the Ethereum infrastructure. Every transaction will be stored in the Blockchain. Forever. You won't need to maintain any infrastructure on your own. On top of that you will be able to monitor every transaction that happened, and detect abnormal activities. For instance if one user manage to steal your wallet private key, you will find evidence directly un the Blockchain.
+
+Regarding DID, you won't need to maintain any infrastructure as well... or maybe you will need. It will depend on your needs. Indeed, No records of your transactions will be made by default. DID will work straight, out of the box. However, if you want to keep track on every transaction, you will have to do it yourself. One could ask: "Why would I need to track every transaction ?". Unlike for Ethereum Smart Contracts, if a user manage to steal your signing private key, you will never be aware of it. Keeping track of every signed document could be a way to counter this issue.
+
+## Environment
+
+Ethereum Smart Contracts are bound to the different Ethereum network. That means if you create a document store on the ropsten network, you won't be able to use the same document store on a different network (mainnet, rinkeby, etc...). It's very convenient for the separation of test and production use cases (like in centralized architecture, when you have staging and production environment). Sometimes you might prefer to have everything working everywhere.
+
+This is where DID come handy. Even if technically a DID could be bound to a specific environment, like `did:ethr:ropsten:0xabcd`, it's easy to use working everywhere, like `did:ethr:0xabc`. Of course while this is true for ethr DID, it might not be true for any DID.
```

Comment:
```suggestion
This is where DID come handy. Even if technically a DID could be bound to a specific environment, like `did:ethr:ropsten:0xabcd`, it's also possible to allow it to be used everywhere, like `did:ethr:0xabc`. Of course while this is true for ethr DID, it might not be true for any DID.
```